### PR TITLE
groff: add output normalization

### PR DESCRIPTION
**Pull request by h4675**
see 'info groff hla' and 'man perlrun' line 201

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/groff.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
 
 `groff -man -T ascii {{manpage.1}} | less`
 
+- Normalize output:
+
+`groff -man -T utf8 {{manpage.1}} | col -bpx | perl -00 -pe '' > {{page.man}}`
```
Comment (CleanMachine1) [inline pages/common/groff.md:17]: This is beginning to become more of a different command. I don't know about this.

Comment (h4675) [reply pages/common/groff.md:17]: When `man` output to file,
1. filter reverse line feeds: `col -bpx` comes from man-db/src/man.c:1616.
2. compress consecutive blank lines: It used a long command of `sed`. In `perl`, `-00` enable paragraph mode.

**Comment by marchersimon** [comment]
Hey @h4675, thanks for your PR, but your changes kind of miss the point of `tldr`. We try to document onlye one command in one page.
But let's see what the others think

**Comment by h4675** [comment]
Now just output normalization, and the `perl` part can also be removed.

**Pull request closed without merging**